resources/views/components/forms/foreign.blade.php

```diff
@@ -19,9 +22,14 @@ class="w-full select2 join-item"
             data-language="{{ LaravelLocalization::getCurrentLocale() }}"
             data-allow-clear="{{ $allowClear ? 'true' : 'false' }}"
             @if (!empty($dropdownParent)) data-dropdown-parent="{{ $dropdownParent }}" @endif
+            :dropdownParent="request()->ajax() ? '#primary-dialog' : null"
     >
         @foreach ($options as $key => $value)
-            <option value="{{ $key }}">{!! $value !!}</option>
+            @if ($multiple)
+                <option value="{{ $key }}">{!! $value !!}</option>
+            @else
+                <option value="{{ $key }}" class="select2-entity" selected="selected">{{ $entity->name }}</option>
```

Comment:
Why $entity->name ? Where is that value coming from magically? Why the select2-entity class? When editing a location, does this change break the parent location dropdown?